packages/service/src/types/rgbpp.ts

```diff
@@ -124,3 +124,24 @@ export interface RgbppApiTransactionRetry {
   success: boolean;
   state: RgbppTransactionState;
 }
+
+export interface RgbppXudtAssetInfo {
```

Comment:
Please consider following the `RgbppApi` naming prefix as the rest of types:
```suggestion
export interface RgbppApiXudtAssetInfo {
```